### PR TITLE
Update node guides for CLI v0.16

### PR DESCRIPTION
**Pull request by fhildeb**
Waiting on:

- [x] https://github.com/lukso-network/tools-lukso-cli/pull/234
- [x] https://github.com/lukso-network/network-configs/pull/131

### CODE REVIEW

**Review by CJ42 — APPROVED** [review]

**Review by Wolmin — APPROVED** [review]

**Review by mxmar — APPROVED** [review]

**Comment by fhildeb** [comment]
@Hugoo @CJ42 LGTM, CLI and network configs have been released 👍🏻

File: docs/networks/advanced-guides/update-clients.md [hunk 1]
```diff
@@ -42,34 +42,75 @@ lukso status
...
-lukso install --geth-tag v1.12
+# https://github.com/ethereum/go-ethereum/releases
+lukso install --geth-tag 1.12.2 --geth-commit-hash bed84606
 
 # Manually overwrite Prysm Version
+# https://github.com/prysmaticlabs/prysm/releases
 lukso install --prysm-tag v4.0.8
 
 # Manually overwrite Lighthouse Version
+# https://github.com/sigp/lighthouse/releases
```
Comment (Hugoo) [inline docs/networks/advanced-guides/update-clients.md:67]: nice to add these links ;)

File: docs/networks/advanced-guides/update-clients.md [hunk 2]
```diff
@@ -42,34 +42,75 @@ lukso status
...
+
+:::
+
+<details>
+  <summary>How to handle Prysm Configuration Errors</summary>
+
+If you updated the network configuration files but are running a Prysm client **below** Version `4.0.8`, your client may not know configuration properties that were introduced in later stages:
+
+```text
+There were some issues parsing the config from a yaml file" error="yaml: unmarshal errors:
```
Comment (Hugoo) [inline docs/networks/advanced-guides/update-clients.md:105]: What is this `"` at the first line? It looks off but maybe this is how the error output looks.

**Review by Hugoo — COMMENTED** [review]
@mxmar could you double check the Clients versions please? If they are compatible.

File: docs/networks/advanced-guides/update-clients.md (hunk 2 above)
Comment (fhildeb) [reply docs/networks/advanced-guides/update-clients.md]: I removed them- the error was copied from the terminal, but they are unnecessary here.

**Comment by fhildeb** [comment]
Client versions are directly taken from: 

https://github.com/lukso-network/tools-lukso-cli/pull/232/

Bildschirmfoto 2024-03-01 um 18 42 04

**Review by mxmar — APPROVED** [review]

**Pull request merged**